app/src/main/java/ml/docilealligator/infinityforreddit/activities/CommentActivity.java

```diff
@@ -233,6 +235,8 @@ public void configureTheme(@NonNull MarkwonTheme.Builder builder) {
                     }
                 })
                 .usePlugin(SpoilerParserPlugin.create(commentColor, commentSpoilerBackgroundColor))
+                .usePlugin(RedditHeadingPlugin.create())
+                .usePlugin(StrikethroughPlugin.create())
```

Comment:
I did a bit more digging/testing and found out that this is applied to both comments which should be parsed and post titles which shouldn't (at least not for spoilers, headings and strikethrough)

So this both fixes and breaks stuff, even if only minor.